code/core/arm9/source/GbaColorCorrection/ColorProfiles.cpp

```diff
@@ -0,0 +1,152 @@
+#include "ColorLut.h"
+
+// Libretro shader values. Credits: hunterk and Pokefan531.
+// https://forums.libretro.com/t/real-gba-and-ds-phat-colors/1540/238
+// Last updated 10-05-2025.
+
+// const ColorProfile PRESET_NAME = 
+//{
+//    {
+//        { [r], [gr], [br] },
+//        { [rg], [g], [bg] },
+//        { [rb], [gb], [b] }
+//    },
+//    luminance
+//};
+
+// libretro GBA AGB-001 color (sRGB).
+const ColorProfile Agb001 =
```

Comment:
```suggestion
static const ColorProfile sAgb001 =
```
Do this for all color profiles in this file